src/database/hooks/useSaveAnonChatHook.ts

```diff
@@ -51,13 +57,15 @@ const useSaveAnonChatHook = () => {
     await chat.save(localDb);
 
     try {
-      object?.members?.forEach((member) => {
+      const members = originalMembers || object?.members;
+      members?.forEach((member) => {
         const saveUserAndChannelListMember = async () => {
           try {
             const userMember = UserSchema.fromInitAnonymousChatAPI(
               member,
               initAnonymousChat?.message?.cid
             );
+
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

This change introduces a fallback to `originalMembers` if it exists, which seems like a good addition for robustness. However, it's crucial to ensure that `originalMembers` is always up-to-date and accurate when this code runs. If there's any chance that `originalMembers` could be stale or incorrect, this could introduce bugs.


<!-- This is an auto-generated comment by OSS CodeRabbit -->